database/nostr-lmdb/Cargo.toml

```diff
@@ -3,20 +3,22 @@ name = "nostr-lmdb"
 version = "0.42.0"
 edition = "2021"
 description = "LMDB storage backend for nostr apps"
-authors.workspace = true
-homepage.workspace = true
-repository.workspace = true
-license.workspace = true
+authors = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
 readme = "README.md"
-rust-version = "1.72.0"
+rust-version.workspace = true
 keywords = ["nostr", "database", "lmdb"]
 
 [dependencies]
-async-utility.workspace = true
+async-utility = { workspace = true }
+flatbuffers = "23.5.26"
```

Comment:
The flatbuffer stuff is used from the nostr-database crate so this dep is unused and can be removed